tests/trainer/test_trainer_cli.py

```diff
@@ -111,28 +98,3 @@ def test_argparse_args_parsing(cli_args, expected):
     for k, v in expected.items():
         assert getattr(args, k) == v
     assert Trainer.from_argparse_args(args)
-
-
-@pytest.mark.skipif(
-    sys.version_info < (3, 7),
-    reason="signature inspection while mocking is not working in Python < 3.7 despite autospec"
-)
-@pytest.mark.parametrize(['cli_args', 'extra_args'], [
-    pytest.param({}, {}),
-    pytest.param({'logger': False}, {}),
-    pytest.param({'logger': False}, {'logger': True}),
-    pytest.param({'logger': False}, {'checkpoint_callback': True}),
-])
-def test_init_from_argparse_args(cli_args, extra_args):
```

Comment:
will this test get removed?

Reply:
Yeah I moved the init_from_argparse function completely so I don't think there's anything to test.

Reply:
where did it get moved? sry but I don't get it. 
These tests were part of a bugfix and are essential to make sure it works.

Reply:
Ahh sorry, you're totally right, this test shouldn't be removed. I was getting it mixed up with the one I removed above. Thanks for catching this!